libs/velo-external-db-core/test/drivers/data_provider_test_support.ts

```diff
@@ -14,6 +17,11 @@ export const givenListResult = (entities: any, forCollectionName: any, filter: a
     when(dataProvider.find).calledWith(forCollectionName, filter, sort, skip, andLimit, projection)
                            .mockResolvedValue(entities)
 
+export const givenItemsById = (entities: Item[], forCollectionName: any, sort: any, skip: any, andLimit: any, projection: any) =>
+    entities.forEach( (entity) => when(dataProvider.find).calledWith(forCollectionName, getByIdFilterFor(entity._id!), '', 0, 1, projection)
+                                                         .mockResolvedValue([entity])
+    )
+
```

Comment:
use the variables sort,skip, andLimit, and maybe change the andLimit to limit because it's not last anymore